pkgs/applications/misc/gnuradio/wrapper.nix

```diff
@@ -1,10 +1,18 @@
-{ stdenv, gnuradio, makeWrapper, python, extraPackages ? [] }:
+{ stdenv, buildEnv, pythonPackages, gnuradio, makeWrapper, python, extraPackages ? [] }:
 
 with { inherit (stdenv.lib) appendToName makeSearchPath; };
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
```

Comment:
I think we should replace this whole derivation with `python.buildEnv`. We can improve that function by having a `makeWrapperArgs ` parameter, which is a list of arguments passed along to `makeWrapper`.

Reply:
Is it possible (does it make sense) to use `toPythonModule` on the gnuradio modules?

Reply:
I agree that using python.buildEnv would be better, but I don't offhand know if the C++ side would be handled well. There is more work that needs to be done, but I wanted to go in smaller steps and prevent breaking the reverse dependencies.

Yes, toPythonModule builds the proper PYTHONPATH and also allows anyone loading this in nix-shell to use those modules.

Reply:
@FRidh do you suggest modifying python.buildEnv?

What is next? 

We should also bump versions to v3.7.13.4 ( #47491  )

Reply:
version bump merged. @FRidh what's next? This shouldn't be critical, just helps us with modules like gr-iio which abuse the gnuradio python namespace.

Reply:
> do you suggest modifying python.buildEnv?

Yes. Let me do this tonight if I don't forget, or open a PR.